tests/golden/control-plane/appcat/appcat/20_xrd_vshn_keycloak.yaml

```diff
@@ -5316,6 +5316,16 @@ spec:
                             ClaimName specifies the name of the instance you want to restore from.
                             The claim has to be in the same namespace as this new instance.
                           type: string
+                        claimType:
```

Comment:
I think this should not be part of this PR right? Also the comment is very vague, I still don't understand what is claim type...

Reply:
This automatically made its way into this after `make gen-golden-all`
I guess I can just leave this as it is?